### PR TITLE
Add suffixes to typos from "ma" to "me"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32229,16 +32229,26 @@ lveo->love
 lvoe->love
 Lybia->Libya
 maake->make
+maange->manage
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32232]: Perhaps `mange` too, although this is a rare word.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -32255,6 +32265,11 @@ macors->macros
 macpakge->package
 macroses->macros
 macrow->macro
+macth->match, math,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32268]: Perhaps `Mach` as well?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -32360,6 +32394,10 @@ mamages->manages
 mamaging->managing
 mamal->mammal
 mamalian->mammalian
+mamber->member
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32397]: Also `amber`, perhaps even `camber`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -32404,12 +32442,20 @@ manetainer->maintainer
 manetainers->maintainers
 manetaining->maintaining
 manetains->maintains
+mangae->manage
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32445]: Also `manga` and `mange`, and `mangle` although the keys **A** and **L** are far apart?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -32404,12 +32442,20 @@ manetainer->maintainer
...
 mangaed->managed
 mangaement->management
 mangaer->manager, manger,
 mangaers->managers, mangers,
+mangaes->manages, mangoes,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32450]: Also `mangas`, and even `mangles`?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -32909,9 +33007,15 @@ memwar->memoir
 memwars->memoirs
 memwoir->memoir
 memwoirs->memoirs
+menaing->meaning, menacing,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33010]:
```suggestion
menaing->meaning, menacing, mending,
```

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32232]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32397]: I'll add `amber`. I couldn't find `camber`.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32445]: I'll add `manga`. I couldn't find `mange` and `mangle`.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32450]: I'll add `mangas`.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**